ui/src/plugins/vuex-persistedstate.js

```diff
@@ -0,0 +1,53 @@
+import Account from "../models/Account";
+import { Locale } from "../models/Locale.ts";
+import { cloneDeep } from "lodash";
+
+export const objectMap = {
+  Account: Account,
+  Locale: Locale
+};
+
+export const persistedStateOptions = {
+  overwrite: true,
+  getState(key, storage) {
+    let value, states;
+    value = storage.getItem(key);
+    states = value ? JSON.parse(value) : undefined;
+    for (let state in states) {
```

Comment:
One potential problem is that it does not encode/restore nested class.
We can create some sort of recursive function to address this issue, but it depends on whether we need it..